code/obj/item/clothing/helmets.dm

```diff
@@ -42,7 +42,7 @@
 
 			if(material.hasProperty("density"))
 				var/prot = round(material.getProperty("density") / 20)
-				setProperty("meleeprot_head", prot)
+				setProperty("meleeprot_head", 2+prot)
 			else
 				setProperty("meleeprot_head", 2)
```

Comment:
Material can never be worse than base helmet. 😦

Reply:
Bit confused on the question. But from what I understand, the base helmet (non-matsci version) has a melee protection of 4, which in inherits from helmet:
![image](https://user-images.githubusercontent.com/84296283/118552974-ae437580-b72d-11eb-86fd-ac7e9955d7b5.png)

The else case, gets used in the case an alloy doesn't have a density property, which in my mind sounds like it has a density property of 0. This could happen if you try making a spacesuit out of something like cerenkite, pizza, glass, cobryl, or negative matter.

So if you had an alloy that had 0 density, you'd get a worse version compared to the base helmet (melee prot 2 vs 4)

Reply:
This only really stands out due to inconsistency between the helmet vs suit calculation.

The default value of the density property is listed as 20.  So I'm not sure if the assumption of 0 is correct here, _though I had that same argument myself when trying to figure out how to calculate Seebeck coefficient for the TEG semiconductor_.  But the math doesn't really pan out that way either...

Reply:
Discussed over Discord with Azrun.
> In a nutshell: I'm not a fan of the mess that was there before you got there.
[mat_properties.dm:124] seems to show it [properties] have a default_value of 20, but :person_shrugging:
Reverse calculation suit density = 26, helmet=40

Main issue is that the equations and default_value don't really play well, if I were to use the `default_value = 20` for materials that have no density property, that would end up buffing the helmet and nerfing the suit protection. I feel like updating it to use those values wouldn't really make sense, so I'll keep them as-is unless there's more opinions on it.

Main hope for this change is to make it so that the reward vs effort is more in-line with how hard it'd be to make a non-matsci space suit and make the reward scale better for higher / lower values.